src/sjur_klemetsen_ex/pa02/snakes_simulations.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Sjur Spjeld Klemetsen, Ola Flesche Hellenes'
+__email__ = 'sjkl@nmbu.no, olhellen@nmbu.no'
+
+import random as rd
+
+
+class Board:
+    def __init__(self, *args, **kwargs):
+        self.board = args
+        self.ladder = kwargs
```

Comment:
It's better to define the required and optional arguments, cause it makes the class easier to be used later